docs/howto/write-unit-tests-for-a-charm.md

```diff
@@ -162,6 +162,50 @@ def test_charm_runs(my_charm):
 If you use pytest, you should put the `my_charm` fixture in a top level `conftest.py`, as it will likely be shared between all your unit tests.
 ```
 
+## Reusing state
+
+Each test is typically an isolated test of how your charm responds to a single event. If you need to use the same input state for several events, we recommend using a fixture that produces the input state. For example:
+
+```python
+@pytest.fixture
+def state_normal():
```

Comment:
Minor: normal_state would read more naturally, though still makes someone wonder what normal means. I’d personally call it good state or quiescent state or perhaps something precise like leader_configured_state or workload_running_state…

That being said, I’m ok with merging in the current form too.